cmd/containerd-stargz-grpc/main.go

```diff
@@ -55,42 +56,48 @@ const (
-	configPath        = flag.String("config", defaultConfigPath, "path to the configuration file")
-	logLevel          = flag.String("log-level", defaultLogLevel.String(), "set the logging level [trace, debug, info, warn, error, fatal, panic]")
-	rootDir           = flag.String("root", defaultRootDir, "path to the root directory for this snapshotter")
-	detachFuseManager = flag.Bool("detach-fuse-manager", false, "whether detach fusemanager or not")
-	printVersion      = flag.Bool("version", false, "print the version")
+	address      = flag.String("address", defaultAddress, "address for the snapshotter's GRPC server")
+	configPath   = flag.String("config", defaultConfigPath, "path to the configuration file")
+	logLevel     = flag.String("log-level", defaultLogLevel.String(), "set the logging level [trace, debug, info, warn, error, fatal, panic]")
+	rootDir      = flag.String("root", defaultRootDir, "path to the root directory for this snapshotter")
+	printVersion = flag.Bool("version", false, "print the version")
 )
 
 type snapshotterConfig struct {
 	service.Config
 
 	// MetricsAddress is address for the metrics API
-	MetricsAddress string `toml:"metrics_address"`
+	MetricsAddress string `toml:"metrics_address" json:"metrics_address"`
 
 	// NoPrometheus is a flag to disable the emission of the metrics
-	NoPrometheus bool `toml:"no_prometheus"`
+	NoPrometheus bool `toml:"no_prometheus" json:"no_prometheus"`
 
 	// DebugAddress is a Unix domain socket address where the snapshotter exposes /debug/ endpoints.
-	DebugAddress string `toml:"debug_address"`
+	DebugAddress string `toml:"debug_address" json:"debug_address"`
 
 	// IPFS is a flag to enbale lazy pulling from IPFS.
-	IPFS bool `toml:"ipfs"`
+	IPFS bool `toml:"ipfs" json:"ipfs"`
 
 	// MetadataStore is the type of the metadata store to use.
-	MetadataStore string `toml:"metadata_store" default:"memory"`
-	// FuseManagerAddress is address for the fusemanager's GRPC server
-	FuseManagerAddress string `toml:"fusemanager_address"`
+	MetadataStore string `toml:"metadata_store" default:"memory" json:"metadata_store"`
 
-	// FuseManagerPath is path to the fusemanager's executable
-	FuseManagerPath string `toml:"fusemanager_path"`
+	// FuseManagerConfig is configuration for fusemanager
+	FuseManagerConfig `toml:"fusemanager" json:"fusemanager"`
+}
+
+type FuseManagerConfig struct {
+	// EnableFuseManager is whether detach fusemanager or not
+	EnableFuseManager bool `toml:"enable_fusemanager" default:"false" json:"enable_fusemanager"`
+
+	// FuseManagerAddress is address for the fusemanager's GRPC server (default: "/run/containerd-stargz-grpc/fuse-manager.sock")
+	FuseManagerAddress string `toml:"fusemanager_address" json:"fusemanager_address"`
+
+	// FuseManagerPath is path to the fusemanager's executable (default: looking for a binary "stargz-fuse-manager")
+	FuseManagerPath string `toml:"fusemanager_path" json:"fusemanager_path"`
```

Comment:
```suggestion
	Path string `toml:"path" json:"path"`
```

Same for other fields too